### PR TITLE
Replace Deriv logo

### PR DESCRIPTION
**Pull request by thibault-deriv**
Replacing Deriv logo in footer and sponsors page:
- Add new `deriv.svg`
- Remove old `deriv.webp`, `deriv-horizontal-web-dark-100-removebg.png` and `deriv-horizontal-web-dark-100.jpeg`
- Update `t/static-files.t` to match added/removed static files

### Preview:
metacpan-preview-deriv-logo-change

### CODE REVIEW

File: t/static-files.t [hunk 1]
```diff
@@ -354,9 +354,8 @@ ee9ccebabfe78643470da003d83e6b61608979f6  root/static/images/sponsors/booking.pn
 12036785cf973c7340818e0837db942830dbf01e  root/static/images/sponsors/control-my-id.png
 8d0a1c548cae06f8d384c7619ee55baabcc27386  root/static/images/sponsors/cpanel.png
 0f8418f040c5504fa69386bafcf7046f11c37503  root/static/images/sponsors/dealspotr.png
-e08d05b3ac61f49694c04540eb4f7ee70a8e6894  root/static/images/sponsors/deriv-horizontal-web-dark-100-removebg.png
-ca16d4cefecab05b457f5ed88415d28f3cce3832  root/static/images/sponsors/deriv-horizontal-web-dark-100.jpeg
-940396b51a065d9580f887d24fd7f5ebf1bd236a  root/static/images/sponsors/deriv.webp
+6f5474eb784a5e9c7498c7a5d773b6ae7b9b5c2d  root/static/images/sponsors/deriv.svg
+ad1e76a2ce9957e0a3725039dbdd1cca9c1a979a  root/static/images/sponsors/.deriv.svg.swp
```
[the remark this answers is not in the document]
Comment (thibault-deriv) [reply t/static-files.t]: :grimacing:

File: t/static-files.t [hunk 2]
```diff
@@ -371,13 +369,14 @@ ddc82658ac274d981a7e3159fc4ed452166f73a4  root/static/images/sponsors/idonethis.
 2b7374c1c82ad51415dacd32d033b119fcfa9cba  root/static/images/sponsors/open-cage.svg
 7f77555505dfcb1a2c3b47bec95062fe2dd7a5da  root/static/images/sponsors/panopta.png
 fb43b99a721b83aeebf528977f3df7083a69289e  root/static/images/sponsors/perl-careers.png
-1eacc5aedbbaeb388801652ecc48c76fce49d530  root/static/images/sponsors/perl-services.svg
 5cdd9d7bda9936c0ab678063a93df341fd37acb1  root/static/images/sponsors/perl_logo.png
+1eacc5aedbbaeb388801652ecc48c76fce49d530  root/static/images/sponsors/perl-services.svg
```
Comment (haarg) [inline t/static-files.t:373]: A bunch of these position changes are due to sort being locale dependent. Could you maybe remove all of the changes except for the ones for the Diriv logo itself?

In the future, there will be a better way to update this file.

Comment (thibault-deriv) [reply t/static-files.t]: OK :+1:

**Review by haarg — APPROVED** [review]

**Comment by haarg** [comment]
Thanks!

**Comment by thibaultduponchelle** [comment]
It’s you guys who deserve a thank you :smile:

**Pull request merged**